themes/bootstrap3/js/cart.js

```diff
@@ -242,6 +245,7 @@ VuFind.register('cart', function Cart() {
     // Search results
     _registerUpdate();
     $("#updateCart, #bottom_updateCart").popover({
+      title: VuFind.translate('Information'),
```

Comment:
Perhaps, I thought about it too, but somehow ended up with the vague "Information". I think it might make sense to also turn the messages into bullets if there are more than one. I'll test a bit.

Reply:
Title changed and bullets added when there are more than one message.